Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -376,6 +380,28 @@
                     return false;
                 }
 
+                //GetEncryptionKey(solution.Guid.ToString());
+                if (solution.FetchEncryptionKey())
+                {
+                    if(solution.ValidateKey())
+                    {
+                        IsEncryptionKeyValid = true;
+                    }
+                }
+                //Solution = solution;
+                if (!IsEncryptionKeyValid)
```

Comment:
Is this flag used in other places as well ? if not then can be removed and directly use solution.ValidateKey() here

Reply:
flag is removed